src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/MainAccordion.tsx

```diff
@@ -42,16 +43,28 @@ type Props = {
   updateEntityUI: (uIstate: { [key: string]: JSONValue }) => void;
 };
 
+enum AccordionSectionID {
+  FUNCTION = 'function',
+  SEQUENCE = 'sequence',
+  OTHER_DATA_SETS = 'other_data_sets'
+}
```

Comment:
Remember we talked about enums, and how we might be using them too eagerly, back when we were discussing a PR about species stats?

Here's a [quick example](https://www.typescriptlang.org/play?#code/C4TwDgpgBAggxnA9gJwCYEtEDsDKE7CZYCSAIlALxQDkAZgK5YFHVQA+NAzhAI70RMIrDtUTAAFhGQB9VAENgc6d2CdqAbgBQ2pFk7AoEAB5g5WVBFQAFMxAA2nAFywEKDNjzNsZANoBdSigfOkYvLGo-LV19QxMzC2tbBwAmZ3gkNCJPQm9Sf0Dg7j4BOCEAGhopZBQI9SgAenqofWR0LABzTigsMSg2uDt6BL6sKAloRiIx8Gg5ZAFqAzk7O0QAd0sdbBj0tyJiYAgAW0CACjBqsCcoAG8oenp0VDTXTI98HJJyAF8ASkoAHzdegrLTaXZvEiHI6nO4PJ7OEJMT6sP5aCHuKHHWH3R7PSrIarIVG-OqNZrAVodbq9fqDYZte5YKagSB9Lo9JYrdaWIA) in typescript playground of achieving the same without an enum (compare it with [example](https://www.typescriptlang.org/play?#code/KYOwrgtgBAggxnA9gJwCYEtEgMrDgF0xAEkARKAbwCgooAxAVQDkBhAFWIHkmoBeKAOQAzMCAJEBAGhpRsAUQCKDOazl9BAZ2ABHMKDjApMzmwAScgEoB9UjDYwr8ttnUDE+ABbBkV1AEN8PystfA0BKgBfKmiqJBANfChgAA8ABz8QVGBUAAUM4AAbDXUAbXgkNCJccSwyADpGVg5uAF1ouITYBBQMWvxgaH4AClTkRFSNAC5KKDAwdFRp8p6qvEJa8giASj4APihwAoKAbhjlyr6BoYpZ+cWuit6cNaJ6xvYuHm3joA) with enums).

The reasons I am more than a bit reluctant to use enums are because:
- It's an old typescript feature, back when they did not hold as priority to conform to standard javascript syntax as much as possible. Now they are trying not to introduce keywords that aren't in JS spec
- It's clunky when it gets transpiled into js. This enum, for example:

```
enum AccordionSectionID {
  FUNCTION = 'function',
  SEQUENCE = 'sequence',
  OTHER_DATA_SETS = 'other_data_sets'
}
```

will get [transpiled](https://www.typescriptlang.org/play?#code/KYOwrgtgBAggxnA9gJwCYEtEgMrDgF0xAEkARKAbwCgooAxAVQDkBhAFWIHkmoBeKAOQAzMCAJEBAGhpRsAUQCKDOazl9BAZ2ABHMKDjApMzmwAScgEoB9UjDYwr8ttnUDE+ABbBkV1AEN8PystfA0BKgBfIA) into this:

```
"use strict";
var AccordionSectionID;
(function (AccordionSectionID) {
    AccordionSectionID["FUNCTION"] = "function";
    AccordionSectionID["SEQUENCE"] = "sequence";
    AccordionSectionID["OTHER_DATA_SETS"] = "other_data_sets";
})(AccordionSectionID || (AccordionSectionID = {}));
```

and there is a special syntax `const enum` (which we aren't using) to [make the js footprint smaller](https://www.typescriptlang.org/play?#code/MYewdgzgLgBApmArgWxgQWKATgEwJbgDKcwUBYAkgCIwDeAUDDAGICqAcgMIAqFA8uxgBeGAHIAZojClyogDSMYhAKIBFVsq7LhYiHACOiBMDjzFfbgAllAJQD6VNNzR2V3QjtEgoACzhY7HABDKCC7PSgIUXoAXyA):

```
const enum AccordionSectionID {
  FUNCTION = 'function',
  SEQUENCE = 'sequence',
  OTHER_DATA_SETS = 'other_data_sets'
}
```
With a [simple union of strings](https://www.typescriptlang.org/play?#code/C4TwDgpgBAggxnA9gJwCYEtEDsDKE7CZYCSAIlALxQDkAZgK5YFHVQA+NAzhAI70RMIrDtUTAAFhGQB9VAENgc6d2CdqAbgBQQA) this problem doesn't even exist.

Reply:
Thanks for explaining it clearly 👍 
I have replaced `enum` with `type AccordionSectionID = 'function' | 'sequence' | 'other_data_sets';`